Automattic/wp-calypso
# Subscriber Details: Unify remove subscriber mutations

Closes https://github.com/Automattic/wp-calypso/issues/78782

## Proposed Changes

* Unifies remove subscriber mutations
* Invalidate details cache if the flag is true
* Rename button label from Unsubscribe to Remove

## Testing Instructions

* Apply this PR to your local
* Go to http://calypso.localhost:3000/subscribers/{site-slug}?flags=subscribers/view-subscriber-details
* Test the Subscribers List for regressions
* Click on the Ellipsis button on a subscriber and see if the Remove button is displayed
* Click on View button and verify if you are redirected to the details page with the current page on the URL
* Click on Remove on header menu
* You should be redirected to the page where you were and the subscribers list should be updated 

<img width="264" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3113712/f1ceee99-d562-49eb-aaf7-95e4b0baca95">

## Pre-merge Checklist
